imports/client/components/FancyEditor.tsx

```diff
@@ -126,19 +128,36 @@ export const MentionSpan = styled.span<{
   display: inline-block;
   overflow-wrap: break-word;
   border-radius: 4px;
-  color: #4649ef;
-  background-color: #ced0ed;
+  color: ${({ theme }) => theme.colors.mentionSpanText};
+  background-color: ${({ theme }) => theme.colors.mentionSpanBackground};
   ${({ $isSelf }) =>
     $isSelf &&
     css`
-      background-color: #4649ef;
-      color: #fff;
+      background-color: ${({ theme }) => theme.colors.mentionSpanBackground};
```

Comment:
Consider preserving an emphasized coloration for mentions that refer to the logged-in user (someone was highlighting you, in particular).  This styling (and the line that follows) basically undoes the work done to identify `$isSelf` by making it styled exactly the same.